Philips.CodeAnalysis.Test/Verifiers/AssertDiagnosticVerifier.cs

```diff
@@ -20,14 +21,14 @@ protected void VerifyError(string methodBody, string expectedDiagnosticId)
 					Severity = DiagnosticSeverity.Error,
 					Location = new DiagnosticResultLocation("Test0.cs", null, null),
 				};
-			VerifyDiagnostic(test, expected);
+			await VerifyDiagnostic(test, expected).ConfigureAwait(false);
 		}
 
-		protected void VerifyNoError(string methodBody)
+		protected async Task VerifyNoErrorAsync(string methodBody)
```

Comment:
This method has the Async postfix, please remove it

Reply:
done